src/store/metadata/stub/content/editions/versions/apprenticeships-achievements_2023-05.json

```diff
@@ -0,0 +1,88 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05/versions",
+    "@type": "hydra:Collection",
+    "title": "Apprenticeships achievements",
+    "versions": [
+        {
+            "@type": [
+                "dcat:Distribution",
+                "csvw:Table"
+            ],
+            "@id": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05/versions/1",
+            "identifier": "1",
+            "issued": "2023-05-17T12:25:28Z",
+            "title": "Apprenticeships achievements",
+            "summary": "Apprenticeships achieved by adults aged 16+ based on home address, rate per 100,000 population, England, academic years 2017/18 to 2022/23.",
+            "description": "This dataset shows apprenticeships achieved by adults aged 16+ based on home address, rate per 100,000 population, in England, for full academic years 2017/18 to 2022/23.",
+            "download_url": "https://staging.idpd.uk/datasets/apprenticeships-achievements/editions/2023-05/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+              "columns": [
```

Comment:
Comparing this to the dataset it appears there are some undefined columns, such as: mission, polarity, notes.

Reply:
The undefined columns - mission, polarity, notes, etc are set to `false` on https://github.com/GSS-Cogs/family-ess/blob/main/apprenticeships-achievements/apprenticeships-achievements.json

So, how should we define and map the columns on the editions and versions file for idpd-api project?